### ISSUE TITLE
FEVER reproduction docs bug

### ISSUE THREAD

**Issue by lintool** [castorini/anserini#1551]
Hey @kelvin-jiang - per https://github.com/castorini/anserini/blob/master/docs/experiments-fever.md

Running:

```
python src/main/python/fever/tune_bm25.py \
 --runs_folder runs/fever-bm25 \
 --index_folder indexes/fever/lucene-index-fever-paragraph \
 --queries_file collections/fever/queries.paragraph.train-subset.tsv \
 --qrels_file collections/fever/qrels.paragraph.train-subset.tsv
```

Got:

```
trec_eval: illegal measure ' recall'
Traceback (most recent call last):
  File "src/main/python/fever/tune_bm25.py", line 77, in <module>
    evaluate_runs(args)
  File "src/main/python/fever/tune_bm25.py", line 50, in evaluate_runs
    '-m map'])
  File "/anaconda3/envs/python36/lib/python3.6/subprocess.py", line 356, in check_output
    **kwargs).stdout
  File "/anaconda3/envs/python36/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['tools/eval/trec_eval.9.0.4/trec_eval', 'collections/fever/qrels.paragraph.train-subset.tsv', 'runs/fever-bm25/run.fever.bm25.k1_0.6.b_0.4.txt', '-m recall.100', '-m map']' returned non-zero exit status 1.
```

Any ideas?

**Comment by lintool** [comment]
So, changing to this works:

```
        results = subprocess.check_output(['tools/eval/trec_eval.9.0.4/trec_eval',
                                           '-m', 'recall.100',
                                           '-m', 'map',
                                           args.qrels_file,
                                           run_file])
```

That is, `'-m recall.100'` -> `'-m', 'recall.100'`.

Must be platform specific... I'm on macOS, @kelvin-jiang had it passing on Linux.

According to https://docs.python.org/3/library/subprocess.html:

<img width="806" alt="Screen Shot 2021-05-30 at 3 33 59 PM" src="https://user-images.githubusercontent.com/313837/120117819-16964c00-c15d-11eb-9844-bdd7168cf2a4.png">

**Comment by lintool** [comment]
Closed by #1552
